docs/hub/model-repos.md

```diff
@@ -310,6 +338,6 @@ Keep in mind that this isn't an exact number, because there are other factors th
 To add **Carbon Emissions** metadata to your models:
 
 1. If you are using **AutoNLP**, this is tracked for you 🔥
-2. Otherwise, use a tracker like  [`codecarbon`](https://github.com/mlco2/codecarbon/) in your training code, then specify `co2_eq_emissions: 1.2345` in your model card metadata, where `1.2345` is the emissions value in **grams**.
+2. Otherwise, use a tracker like  Code Carbon in your training code, then specify `emissions: 1.2345` in your model card metadata, where `1.2345` is the emissions value in **grams**. 
```

Comment:
```suggestion
2. Otherwise, use a tracker like  Code Carbon in your training code, then specify `co2_eq_emissions.emissions: 1.2345` in your model card metadata, where `1.2345` is the emissions value in **grams**. 
```